otherlibs/stdlib_alpha/capsule.ml

```diff
@@ -357,8 +359,10 @@ module Rwlock = struct
     ; rwlock : Rw.t
     ; mutable poisoned : bool
     }
+  [@@unsafe_allow_any_mode_crossing]
 
   type packed : value mod portable uncontended = P : 'k t -> packed
+  [@@unsafe_allow_any_mode_crossing]
```

Comment:
Like the other `packed`, this one can be removed once we can handle GADT constructors.